openedx_events/learning/signals.py

```diff
@@ -117,3 +123,15 @@
         "cohort": CohortData,
     }
 )
+
+
+# .. event_type: org.openedx.learning.discussions.configuration.change.v1
+# .. event_name: COURSE_DISCUSSIONS_UPDATED
+# .. event_description: emitted when the configuration for a course's discussions changes in the course
+# .. event_data: CourseDiscussionConfigurationData
+COURSE_DISCUSSIONS_UPDATED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.discussions.configuration.change.v1",
```

Comment:
I know we are only migrating this OpenEdxPublicSignal from the core platform here, but according to the naming rules, the name should use `changed` as the signal fire when the change has been completed.

@mariajgrimaldi what do you think about this? does changing this now break havoc on everything?

@xitij2000, you are the original author of the signal, are you ok with us updating this?

Reply:
Yes! Well spotted @felipemontoya.

I'm wondering, should we give examples in the ADR of event_name -> event_implemented_name transformation?

Reply:
Sorry for the delay in responding. Currently, there is only one consumer of this event, but for changing the name, what could be side-effect? These are django signals, so I don't expect there will be a situation where the signal dispatcher and handler are running two different versions of the code. 

I feel it should be safe to moving and renaming.

Reply:
Our plan is to merge this PR and immediately open a new one with the update in edx-platform, so there's no version mismatch of any kind

Reply:
That's correct!!!
So, @felipemontoya what do you think about feed back? Any other suggestion?

Reply:
I say let's move forward with the changes :)